frontend/core-ui/src/modules/notification/my-inbox/components/contents/system/WelcomeMessage.tsx

```
@@ -1,130 +1,272 @@
+  <motion.div
+    initial={{ opacity: 0, y: 20 }}
+    animate={{ opacity: 1, y: 0 }}
+    transition={{ delay: 1.0 }}
+  >
+    <h3 className="text-base font-medium text-muted-foreground text-center mb-4">
+      Contact Us:
+    </h3>
+    <div className="flex flex-wrap justify-center gap-2">
+      {Socials.map((item) => {
+        return (
+          <Link to={item.url}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)